bundles/org.openhab.core.automation.module.script.rulesupport/src/main/java/org/openhab/core/automation/module/script/rulesupport/loader/ScriptFileWatcher.java

```diff
@@ -111,6 +103,21 @@ public void deactivate() {
         super.deactivate();
     }
 
+    @Override
+    public void activate() {
+
```

Comment:
If  you have to touch it again, please remove this empty line too.

Reply:
I've removed this. I also thought I'd take a quick look at nullness annotations in case it was a trivial addition, and it was, so I added these too.